tests/test_wave_05.py

```diff
@@ -123,28 +148,30 @@ def test_delete_goal(client, one_goal):
     # Check that the goal was deleted
     response = client.get("/goals/1")
     assert response.status_code == 404
+    #my test:
+    assert Goal.query.get(1) == None
```

Comment:
Great start! It would also be a good idea to get the `response_body` from the new response to make sure it has the message you're expecting.